platform/security/sts/security-sts-server/src/main/java/ddf/security/sts/EHCacheTokenStore.java

```diff
@@ -71,19 +70,23 @@ public EHCacheTokenStore(String key, Bus b, URL configFileURL) {
 
     Cache newCache = new RefCountCache(cc);
     cache = cacheManager.addCacheIfAbsent(newCache);
-    synchronized (cache) {
+    lock.lock();
```

Comment:
@roelens8 Would you please review the changes I made to this class? I had to update it for the removal of FindBugs and noted this sonarqube issue while reviewing my changelist. The lock usage is correct now, but I'm not sure it matches the semantics you were going for.

Reply:
👍